### DISCUSSION TITLE
Multi Webpack loader

### DISCUSSION THREAD

**Discussion by 68ociredef** [ocombe/ng2-translate#1389] [Show and tell]
Hey folks!, 
Working on my company project I had to make a loader that loads translations using webpack imports of json files.
This is the link https://github.com/68ociredef/ngx-translate-multi-webpack-loader.
I hope that it is utile :)

**Comment by rbalet** [comment]
Multi loader will now be available in the v17.1. 

Thanks for your help though. 
Happy coding
